lit/docs/install/worker.lit

```diff
@@ -76,53 +77,51 @@ decide much on its own.
-    re-created (i.e. fresh VM/container and all processes were killed), it should
-    be brought back with an empty disk.
+    \bold{Bandwidth usage}: again, almost entirely subject to pipeline
+    workloads. Expect spikes from periodic checking, though the intervals
+    should spread out over enough time. Resource fetching and pushing will also
+    use arbitrary bandwidth.
 
-    Highly available: not applicable. Workers are inherently singletons, as
-    they're being used as drivers running entirely different workloads.
+    \bold{Disk usage}: arbitrary data will be written as builds run, and
+    resource caches will be kept and garbage collected on their own life cycle.
+    We suggest going for a larger disk size if it's not too much trouble. All
+    state on disk must not outlive the worker itself; it is all ephemeral. If
+    the worker is re-created (i.e. fresh VM/container and all processes were
+    killed), it should be brought back with an empty disk.
 
-    Horizontally scalable: yes; workers directly correlate to your capacity
-    required by however many pipelines, resources, and in-flight builds you want
-    to run. It makes sense to scale them up and down with demand.
+    \bold{Highly available}: not applicable. Workers are inherently singletons,
+    as they're being used as drivers running entirely different workloads.
 
-    Outbound traffic:
+    \bold{Horizontally scalable}: yes; workers directly correlate to your
+    capacity required by however many pipelines, resources, and in-flight
+    builds you want to run. It makes sense to scale them up and down with
+    demand.
 
+    \bold{Outbound traffic}:
     \list{
-      external traffic to arbitrary locations as a result of periodic resource
+      External traffic to arbitrary locations as a result of periodic resource
       checking and running builds
     }{
-      external traffic to the \code{web} node's configured external URL when
+      External traffic to the \code{web} node's configured external URL when
       downloading the inputs for a \reference{fly-execute}
     }{
-      external traffic to the \code{web} node's TSA port (\code{2222}) for
+      External traffic to the \code{web} node's TSA port (\code{2222}) for
       registering the worker
     }
 
-    Inbound traffic:
-
+    \bold{Inbound traffic}:
     \list{
-      various connections from the \reference{web-node} on port \code{7777}
-      (Garden), \code{7788} (BaggageClaim), and \code{7799} (garbage collection)
-    }{
-      repeated connections to \code{7777} and \code{7788} from the
-      \reference{web-node}'s TSA component as it heartbeats to ensure the worker
-      is healthy
+      From the \reference{web-node} on port \code{7777} (Garden) and
+      \code{7788} (BaggageClaim). These ports do not need to be exposed, they
+      are forwarded to the web node via the ssh connection on port 2222.
     }
   }
 }
```

Comment:
There's inbound + outbound traffic to other worker nodes if p2p streaming is enabled